db/models.py

```diff
@@ -50,3 +53,54 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
Ensure that the `User` model is correctly referenced in the `Order` model's `ForeignKey`. Since the `User` model is defined in the same file, the string reference 'User' is appropriate.